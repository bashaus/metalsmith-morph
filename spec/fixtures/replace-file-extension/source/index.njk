---
title: Lorem ipsum
---

<h1>{{ title }}</h1>

Body copy
