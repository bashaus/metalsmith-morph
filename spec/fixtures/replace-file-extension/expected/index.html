
<h1>Lorem ipsum</h1>

Body copy
